.setup/distro_setup/ubuntu/22.04/setup_distro.sh

```diff
@@ -20,6 +20,7 @@ fi
 # Need to change this otherwise it will hang the script in interactive mode
 sed -i "s/#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf
 
+add-apt-repository ppa:ondrej/php
```

Comment:
Does this need a `-y`?

Reply:
I don't think it needed it, but I added it just in case